MDS/Collivery/Model/MdsCollivery.php

```diff
@@ -286,20 +337,24 @@ public function getLocationTypes()
             return $this->cache->get('collivery.location_types');
         } else {
             try {
-                $result = $this->client()->get_location_types($this->token);
-            } catch (SoapFault $e) {
-                $this->catchSoapFault($e);
+                $param =["api_token" => ""];
+                if($this->token!=null)
+                    $param["api_token"]=$this->token;
```

Comment:
Always add `{ .. }` to `if` (and all) statements 
Please see https://www.php-fig.org/psr/psr-1/ and https://www.php-fig.org/psr/psr-12/